### DISCUSSION TITLE
The user interface translation into Estonian

### DISCUSSION THREAD

**Discussion by vroosioks** [SafeExamBrowser/seb-win-refactoring#716] [Q&A] [answered]
I found this conversation [https://sourceforge.net/p/seb/discussion/844844/thread/0b4f525ee9/?limit=25#37a2](https://sourceforge.net/p/seb/discussion/844844/thread/0b4f525ee9/?limit=25#37a2) and would like to ask is this possibility to translate the user interface into another language still exist?

**Comment by dbuechel** [comment]
Yes, if you'd like to contribute a new language, please feel free to translate the following document: https://github.com/SafeExamBrowser/seb-win-refactoring/blob/master/SafeExamBrowser.I18n/Data/en.xml. SEB for macOS and iOS require a different translation, for those you'd best ask in the respective repository here on GitHub.

**Reply by vroosioks** [reply, marked as answer]
I translated the en.xml file into Estonian. It can be downloaded here: [https://github.com/vroosioks/SEB/blob/main/et.xml](https://github.com/vroosioks/SEB/blob/main/et.xml)

**Reply by dbuechel** [reply]
Thanks a lot for your contribution, it is highly appreciated! We'll integrate the translation into the upcoming release version 3.6.0.

**Reply by dbuechel** [reply]
The translation can now be tested with the latest beta build of version 3.6.0: https://sebdev-let.ethz.ch/api/buildjobs/solmaainb2ccfaoq/artifacts/SEB_3.6.0.632_SetupBundle.exe. Thanks again for your contribution!
